buildpipeline/pipeline.json

```diff
@@ -20,7 +20,7 @@
             "PB_DockerTag": "centos-7-d485f41-20173404063424",
             "PB_BuildArguments": "-buildArch=x64 -Release -stripSymbols",
             "PB_BuildTestsArguments": "-buildArch=x64 -Release -SkipTests -Outerloop -- /p:ArchiveTests=true /p:EnableDumpling=true",
-            "PB_SyncArguments": "-p -- /p:ArchGroup=x64 /p:OverridePackageSource=$(PB_AzureFeedUrl)",
+            "PB_SyncArguments": "-p -- /p:ArchGroup=x64 /p:DotNetRestoreSources=$(PB_RestoreSource)",
```

Comment:
@dagood @eerhardt do you guys see any issue with using DotNetRestoreSources to pass this information? I was debating between it and OverridePackageSource an leaned towards DotNetRestoreSources as that is intended to be used as an extension point for passing along a new feed.

Reply:
I have a few minor issues with it, but I think it would work fine.

 1. Currently the API spec says you get a props file with `DotNetRestoreSources` in it as `DotNetRestoreSourcePropsPath`, but technically `DotNetRestoreSources` isn't an API itself. We'd need to promote it.
 2. If you define `DotNetRestoreSourcePropsPath` and `DotNetRestoreSources` at the same time, they won't both be applied (at least, with a reasonable MSBuild implementation). We can just say we won't pass both at the same time.
 3. It implies that it can take multiple semicolon-delimited sources, but in reality build scripts usually choke when you try that from the command line. We can just say that the orchestrator will only pass one source on the command line.

Reply:
I agree with our points but I think we are good as we should be using only one or the other. There is risk that if we pass it as a parameter to msbuild then we might not be able to override it correctly but based on my expectations of it's usage in source-build we will likely pass it as a parameter there as well (although it currently is an env variable).

Reply:
> based on my expectations of it's usage in source-build we will likely pass it as a parameter there as well (although it currently is an env variable).

I haven't seen it as an env variable yet, only inside the `DotNetRestoreSourcePropsPath` XML. If we can slim source-build down to a single feed at some point, we could pass it in with `DotNetRestoreSources`, but for now, `DotNetRestoreSourcePropsPath` exists so we can pass more than one.

Reply:
Your correct I was thinking of DotnetRestorePackagesPath https://github.com/dotnet/source-build/blob/dev/release/2.0/repos/dir.props#L28